enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -10,30 +10,33 @@ import (
 
 	"github.com/coder/coder/v2/coderd/database"
 	"github.com/coder/coder/v2/coderd/database/dbauthz"
+	"github.com/coder/coder/v2/coderd/prebuilds"
 )
 
 var (
```

Comment:
The descriptions for failed, exhausted, actual, and desired are too vague; please add some more detail.

Reply:
👌 perfect, thanks